.gitignore

```diff
@@ -27,3 +27,4 @@ xcuserdata/
 *.playground
 playground.xcworkspace
 timeline.xctimeline
```

Comment:
I don't think we can add this to the git ignore as this file needs to be tracked as the project changes. I believe you changed this to avoid signing changes. I think the way to resolve that, while tracking other project changes, is to model Loops code signing which uses xcconfig files.